src/query/ts/m3db/consolidators/step_accumulator.go

```diff
@@ -104,6 +102,7 @@ func (c *StepLookbackAccumulator) AccumulateAndMoveToNext() []xts.Datapoint {
 	}
 
 	val := c.unconsumed[0]
-	c.unconsumed = c.unconsumed[1:]
+	copy(c.unconsumed, c.unconsumed[1:])
+	c.unconsumed = c.unconsumed[:len(c.unconsumed)-1]
```

Comment:
Can you remove changes to this file from here?  This is coming in:
https://github.com/m3db/m3/pull/2185